java/arcs/core/storage/ReferenceModeStore.kt

```diff
@@ -145,13 +145,15 @@ class ReferenceModeStore private constructor(
         }
     )
 
+    private var containerCallbackToken: Int = -1
```

Comment:
Since this is initialized in the `init`, you can probably make it a `val` and leave the initialization until `init`.  Then you don't have to check if it's equal to -1 in `off`